### ISSUE TITLE
Suggestion strings being encoded incorrectly

### ISSUE THREAD

**Issue by henrebotha** [segabor/Hunspell#3]
Try the following:

``` ruby
sp = Hunspell.new('en_ZA.aff', 'en_ZA.dic')
s = sp.suggest("Skjhd")[0]
# => "Ski\xE2\x80\x99d"
s.encoding
# => #<Encoding:ASCII-8BIT>
s.encode("utf-8")
# => Encoding::UndefinedConversionError: "\xE2" from ASCII-8BIT to UTF-8
```

(I'm using a South African English dictionary from [here](http://extensions.openoffice.org/en/search?query=south+african+english).)

What seems to be happening is that the string is incorrectly encoded as ASCII, when it is in fact UTF-8. Since the UTF code points are not valid ASCII, it throws an error when attempting to convert them from ASCII (supposedly) to UTF.

If I force the encoding, the string renders correctly:

``` ruby
s.force_encoding("utf-8")
# => "Ski’d"
```

**Comment by segabor** [comment]
Thanks for the report. I will provide a fix based on this article http://tenderlovemaking.com/2009/06/26/string-encoding-in-ruby-1-9-c-extensions.html

**Comment by segabor** [comment]
Fix is out with release 0.1.4. Please let me know if it's okay.
